### PR TITLE
Adjusted batch file paramters & attributes as also...

### PR DESCRIPTION
**Pull request by nilskuehme**
suppressing passphrase using :update resource

# Description

Describe what this change achieves

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by damacus** [comment]
Hey, 

thanks for the contribution. Looks like there's a fix needed in your markdown 

```
README.md:57: MD032 Lists should be surrounded by blank lines
```

**Comment by nilskuehme** [comment]
Hey @damacus, 

thanks for the hint. Changed readme.md to match markdown rule MD032.

**Comment by damacus** [comment]
Looks like delivery is failing, due to the updated Rubocop version since we ran this. If you feel up to fixing that, we'd really appreaceate it. Otherwise I'll go do it this afternoon and you can rebase of master once that's in.

**Comment by nilskuehme** [comment]
I am currently unable to attend and need to accept your approach.

**Comment by nilskuehme** [comment]
Hey @damacus, now i do have time to improve that. Wasn't able to find out which Rubocop version currently is beeing used. Do i need to change the syntax of the bash entry `rm -f $TMP_REQUEST` for Rubocop? Can you help me out what to do?

File: recipes/default.rb [hunk 1]
```diff
@@ -105,4 +107,5 @@
   #{gpg_command} --batch --gen-key $TMP_REQUEST
   EOH
   not_if { ::File.exist?("#{node['aptly']['rootDir']}/.gnupg/trustdb.gpg") }
+  rm -f $TMP_REQUEST
```
Comment (damacus) [inline recipes/default.rb]: yeah, you just need to remove this line. It looks like you're trying to run back, but you're actually in a Ruby block!

Comment (nilskuehme) [reply recipes/default.rb]: Makes sense, for somehow i've chosen the wrong line - so i corrected it. The change does work on my setup.

**Comment by damacus** [comment]
And now delivery is failing on some unit tests. 

If you run `chef exec delivery local all` locally you'll see all the failures 👍

**Comment by nilskuehme** [comment]
I never worked with delivery local all. So i've did the kitchen run from the aptly cookbook itself. That worked well:

`-----> Kitchen is finished. (22m22.78s)
`
Saw that the CHANGELOG.md was not updated well. Updated that file.

**Comment by damacus** [comment]
You'll need to run delivery (part of [Chef Workstation](https://downloads.chef.io/chef-workstation/)) to see the failures locally. But right now it looks like you've got some simple fixes in the unit tests to make before we can accept this :)

**Comment by nilskuehme** [comment]
I will do find out how it works and will do these corrections if possible during the weekend.

**Comment by nilskuehme** [comment]
Hey @damacus,

changes have been made from my side. Now the danger stage is only the remaining one. It that something i can solve or is it you need to do on your side?

**Comment by JohnRoesler** [comment]
There is a bug with Danger right now - not a result of your PR.

**Comment by nilskuehme** [comment]
@JohnRoesler does that bug needs to be resolved before you can accept these changes or are you able to accept it anyway? Because during my last PR there were also mistaken tests and it has been accepted and merged into master.

**Pull request merged**